### PR TITLE
Fields to properties for Database classes; for Starcounter/TechComm#340

### PR DESCRIPTION
**Pull request by Mihaiii**
Fields to properties for Database classes; for Starcounter/TechComm#340  and Issue #53 .

I skipped the ones that had `SynonymousTo` attribute and also `Name` from Simplified.Ring1.Something as making them properties would rise [this](https://github.com/Starcounter/level1/wiki/SCERR4070) error.

### CODE REVIEW

File: Ring6/Web/WebMap.cs [hunk 1]
```diff
@@ -11,6 +11,6 @@ public class WebMap : Relation {
         public WebUrl Url;
 
         public string ForeignUrl;
```
Comment (bouguila) [inline Ring6/Web/WebMap.cs]: i think you should also apply the transformation to the field ForeignUrl

Comment (bouguila) [inline Ring6/Web/WebMap.cs]: also do't you think that the field name should have `[SynonymousTo(nameof(Name))]` attribute?

**Comment by Mihaiii** [comment]
Made the change regarding `ForeignUrl` (thanks!).
Regarding the discussion about `Name`'s attribute, in my opinion it's outside the scope of this PR.

**Comment by joozek78** [comment]
It seems that the build is failing, because this branch is based on master. Shouldn't it be merged with develop?

**Comment by Mihaiii** [comment]
The instructions were to merge to master: https://github.com/StarcounterApps/Simplified/issues/53#issue-239140716

**Comment by Mihaiii** [comment]
Sorry, the parent branch was develop. Will change now.

**Comment by Mihaiii** [comment]
Deleted previous comment (I thought that the parent branch was `develop`) as it was indeed based on `master`.

**Comment by warpech** [comment]
`master` branch builds are failing, too: https://github.com/StarcounterApps/Simplified/commits/master, for reasons beyond this PR

**Comment by warpech** [comment]
@bouguila is this good for merging now?

**Comment by bouguila** [comment]
yes looks good now! thanks

**Pull request merged**